packages/story-editor/src/components/canvas/frameElement.js

```diff
@@ -80,41 +90,72 @@ const EmptyFrame = styled.div`
 const NOOP = () => {};
 
 function FrameElement({ id }) {
-  const { isSelected, isBackground, element } = useStory(({ state }) => ({
-    isSelected: state.selectedElementIds.includes(id),
-    isBackground: state.currentPage?.elements[0].id === id,
-    element: state.currentPage?.elements.find((el) => el.id === id),
+  const [isTransforming, setIsTransforming] = useState(false);
```

Comment:
cc @littlemilkstudio Tagging you here just in case to review this file from performance perspective.
If I'm not mistaken you looked into performance of rendering the elements and frames and some point, and looks like there are a bunch of new dependencies added here. (Feel free to ignore if I remember incorrectly)

Reply:
@miina Thanks for tagging. Hmph, yea it looks like this introduces some regressions there.

For reference, most of the perf work I did in here was just around not causing a render for every frame element when only one needed updating. Tested this by doing some simple actions like select, or moving an element.

Looks like this PR introduces a regression here. Here's a picture of the flame graph when selecting a new element:
<img width="681" alt="Screen Shot 2022-03-07 at 10 23 38 AM" src="https://user-images.githubusercontent.com/35983235/157075416-4313763f-a9c3-4ac4-9962-a8f5a6355f98.png">

Reply:
Not able to see much difference in my profiler test. Can you lay down the steps you followed to get to this result?

Reply:
This seems to have improved now

**Before:**

![before_be6a5ba](https://user-images.githubusercontent.com/6297436/157416521-372fca1b-b46b-4f55-8269-73d1814f3d5d.png)


**After:**

![after_be6a5ba](https://user-images.githubusercontent.com/6297436/157416542-52e71634-8c73-416e-ac32-c0099f6e277f.png)


Tested by
- adding multiple kinds of elements.
- selected one of them
- started Profiling
- selected any unselected element
- stoped profiling

Reply:
Results from the `main` branch

![main](https://user-images.githubusercontent.com/6297436/157417226-5fb17abd-278d-4066-a148-7ea5d8fb02fe.png)